src/components/pages/user/profile.tsx

```diff
@@ -298,8 +350,12 @@ const ProfileHeader = styled.header`
   }
 `
 
-const Gray = styled(StyledP)`
+const Aside = styled.aside`
   margin-top: 70px;
-  font-size: 0.9rem;
   color: #777;
+
+  & > p {
+    font-size: 0.9rem;
+    margin-bottom: 20px;
+  }
```

Comment:
try to avoid using selectors, because they are spaghetti ^^

apply classnames directly to corresponding p's (e.g. Roles and line 94)